dockerfiles/docker-compose.yml.template

```diff
@@ -0,0 +1,10 @@
+elasticsearch:
+  image: mozilla/dxr_elasticsearch
+  net: host
+  
+dxr:
+  image: mozilla/dxr_application
+  net: host
+  links:
```

Comment:
Trying to remember back to when I cooked this up, it's been so long!

IIRC, I was trying to not use `net: host` to keep containers talking among themselves over docker's bridge. But I think for doing dev work from outside the container and for ease of testing, the net:host is the better approach so I went with that. Long way of saying, links is redundant, since net: host will ignore them anyway.